airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -14,7 +14,7 @@ java {
 airbyteJavaConnector {
     cdkVersionRequired = '0.23.19'
     features = ['db-sources', 'datastore-postgres']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
In order to properly validate this change, it needs to be applied to at least one connector.

Reply:
yep - this change exactly serves this before releasing cdk - so all tests on postgres would pass with this change. once approved i'll need to release cdk and bump up the required version here too.